src/schema_formats/XML/GMP.xml.in

```diff
@@ -15928,6 +15928,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
         <summary>ID of filter to use to filter reports</summary>
         <type>uuid</type>
       </attrib>
+      <attrib>
+        <name>details</name>
+        <summary>
+          Whether to get the details of the reports including the
+          results, hosts, ports etc.</summary>
```

Comment:
```suggestion
          results, hosts, ports etc.
        </summary>
```

Actually the idea of summary is that it is short and clear, so it should not span multiple line.
Description is for muliple lines.

Reply:
That makes sense but currently the description text is not included in the HTML file for attributes and for me the text fits into one line there at least.